wireless/bluetooth/Kconfig

```diff
@@ -165,6 +165,16 @@ config BLUETOOTH_BUFFER_IRQRESERVE
 		interrupt level.  This setting only needs to be non-zero if your
 		low-level Bluetooth driver needs to do such allocations.
 
+config BLUETOOTH_CNTRL_HOST_FLOW_ENABLE
+	bool "Enable Controller to Host Flow Control"
+	default n if BT_CTRL_HOST_FLOW_CONTROL_SUPPORT_DISABLED
+	default y
```

Comment:
I think you can replace both with just:
```
default !BT_CTRL_HOST_FLOW_CONTROL_SUPPORT_DISABLED
```

This way it will be disabled when BT_CTRL_HOST_FLOW_CONTROL_SUPPORT_DISABLED is enabled